EESSI-pilot-install-software.sh

```diff
@@ -313,7 +313,7 @@ check_exit_code $? "${ok_msg}" "${fail_msg}"
 echo ">> Installing RStudio-Server 1.3.1093..."
 ok_msg="RStudio-Server installed, enjoy!"
 fail_msg="Installation of RStudio-Server failed, might be OS deps..."
-$EB RStudio-Server-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb --robot
+TZ=UTC $EB --from-pr 12544 RStudio-Server-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb --robot
```

Comment:
The `TZ` thing is just precautionary, I've no idea if it is really required

Reply:
It's not required, only needed for very specific cases like rstan (see #79), so please drop it.

Reply:
Removed, however I did find on Google that this can be an issue for RStudio

Reply:
link?

Reply:
Nope, can't find it now, it was mentioned on one of the links I followed related to se-linux and the error that this change solves:
```
rserver[7485]: ERROR system error 13 (Permission denied) [path: /var/run/rstudio-server, target-dir: ]; OCCURRED AT rstudio::core::Error rstudio::core::FilePath::createDirectory(const string&) const src/cpp/shared_core/FilePath.cpp:648; LOGGED FROM: int main(int, char* const*) src/cpp/server/ServerMain.cpp:555
```
something about a timeout of something due to it  not being executable because of se-linux and setting `TZ` as being a way of getting around that

Reply:
Found it https://github.com/rstudio/rstudio/issues/4937#issuecomment-643060730